### PR TITLE
Add support for linking sets through Loadouts

### LINKED ISSUE #4569: Link passive trees, skill sets and item sets
### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Is your feature request related to a problem?

Passive trees and the skills sets added in 2.19.0 aren't connected. Item sets aren't connected either.

### Describe the solution you'd like

I'd like the possibility to link passive trees with skill sets. This would be useful for:

* Having multiple leveling trees tied to different leveling skill sets.
* Reservation. For example, if a passive tree invests in armor, it would be logical to link it to a skill set with determination. If I have an alternative passive tree with a focus on energy shield, maybe I would like to have it linked to a skill set that swaps determination for discipline. Or maybe another tree doesn't invest at all in reservation efficiency and I need to have it linked to a skill set with fewer auras. Not having the skill sets linked to the passive trees makes the tree comparison feature useless in this scenario.

The same is applicable for item sets.

### LINKED ISSUE #7667: Change same named sets together
### Check for duplicates

- [X] I've checked for duplicate open **and closed** issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues?q=is%3Aissue)

### What platform are you running Path of Building on?

Windows

### Is your feature request related to a problem?

No, there's no problem, this is pure QoL.

### Describe the solution you'd like

If there are same named Item set / Skill set / tree then changing one should change all three.

I do not know enough Lua to know how to reach across classes but if there's a way to do this then I imagine making a helper which does all three in a single go and calling it from the below code points should do it:

https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/5c2dd5a963159d6ea13b226ce5809316d2b0f347/src/Classes/SkillsTab.lua#L97

https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/5c2dd5a963159d6ea13b226ce5809316d2b0f347/src/Classes/TreeTab.lua#L41

https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/5c2dd5a963159d6ea13b226ce5809316d2b0f347/src/Classes/ItemsTab.lua#L67

Edit: ah. https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/5c2dd5a963159d6ea13b226ce5809316d2b0f347/src/Classes/SkillsTab.lua#L184

I might attempt a PR in a few hours.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Peechey**
Fixes #7667
Fixes #4569 

### Description of the problem being solved:
This is Subtractem's PR #6659 with my changes on top to account for allowing the same set to be used multiple times. Exact name match still works of course. Links are designated by identifiers within braces, comma-delimited. I JUST got this working so it's messy but we want people looking it over.

- [x] Comments
- [x] Help popup in dropdown to thoroughly explain what the hell kinda system I cooked up or maybe I just update the About > Help?
- [x] SecondaryAscendancy is commented out

### Build:
<pre>eNrtPdty2ziyz6uvYKlqz0tkGwABXnKS3ZLl6ya-jGUnk7y4QBK0mFCkQlK-TGr-_TRASiJlUZZseepkljtbjkSgG0DfuwFQ7_59Pwy1W5GkQRy9b-Nt1NZE5MZeEN28b19dHmxZ7X__q_XunGeDM393HISy5V-tf7xTn7VQ3Irwfds22lrGkxuRfZqg0q8B1YhH2UDE0Qn_FieHsfe-fRpHoq05PPKCbPLNDXmanvKheN_uD7gX37U1nroi8nqzhsskcL-nmUig-4An3IWPH-Xg3XEWn8QedPF5mAK2IQ-ifux-F9lhEo9H79u0rd0G4i7vdHxyfnZxWZpZEJVnBiv7x7vzkD-IpJ_xTEvhz_t2FwjEb8QeH8JfwMbDMaBiNrPRtt7eWQq0O07SbD3I_kgIb9Z52zYty6rrfJ6Ifd8Xbhbcil4SZL0Bj9zZUJhs26wO9jn9T8ZhFozCQHJiAlPX_-gxeoTqOl_GGQ_3zvvTvjoziKVvE0Ysgy6HirMpVC3-z0E22A2BsmuPISGPb6IgE88CPY-DNI6evbIyYO3ieuMwBMVcqe-FSEVyy7OgOql63PHQCaJn0K2bCH7m59J5wb1gnJ6ILBHpFAvZrhWdEx7xXpzO-IrtZV3PRQIGI6tAoCcA-sKNwcaUQXRz2zIMQlcYajGC2jE_Br5YvedayykA1p3N89ax31-139qInzehCzCsq_Xsx-NwxZ5ZybgZVr0i_aj0ZLWGc0_cz4SeLkNY7rkE4XE0Wwa18BKE5Z5LZ3gbS4vwNHGUXdo_Oi9RCGG6bSFq6Toza_3U4CENXB6e8PtgOB6Ca7jk30VU8gx2vYp_DG4GWQQGrg6amtQ06qAPgkTUAjJqW_WWL_SeBzjgcVoHaTPLrDcwQXQEwVHXdccQ5DyUDe5SbZ6JFzHxksDCfSt7H0fubCFkGd6rKFHuohSQLBtBglyA9srYxwnFyjCzYQorsErEkA92I6JixIfVDN9HIdzBIRD6gmdiNW9RCo8stpS6sneZurqxDO8C6mJkrQYxTyhrmyJqGDYxkInQUhe2mGgW21462TXpth-J5OahPwhEOFudrltkFYDJ5Hp8tBKson0ZQZkHGJmrjbmWKJVBF5MGY2pvY4OZBNcS9jNPvNWc3JrTy8NHCVNSQmMbk6UApyK7qMBsYYq2WS0Q9J9q4Qxm-UAAMxXClYQPAB7xaQX-3PK07NCwQepD1Vx-coiy6BC7XpEEJBMA4om5HMdg9bla_E1mauGacN1kGI-TFeIWuYq8c3kR6CmfnKenF8Ibu6vFALshJNhz0182qzBcC6KbZdz9vhd7N2KtQdaCmGbNCrQ_Ho3Askr-r56yysACsqmgFFMuj0LyzmegwWXDhpfFHyvjn3Wex68_GVbND2KuCDE_EiFLI6K5YQheoff8EE_z8wQs5BA8oyqjnMQl17pt01p1PoAMeu0MVwEtTtELfP3vQRhqqfx7zpOJ2dAgKGxrKeipCyDtiQwPg0xosa_tBbeQcGv_GYM4D8s-fqVJncd3QLKBLMelK9iAUm-IWJ8WmoNERH88rIy_0n2lAfYjD2JfUMGVx5iHWGmYrhIW8Hpi-DEYBtkjdbwMhsCrNN3jGde-iTsRXj6MxLEne2hekS994knAo4yoh6pTXnuU3ahlQvKopYIn7uAjyPIBD0MHLJti-fTp-_Yt5-F1BP4XQuZrV8r-tadM8jXuaPn_UavaaZyk5VYpTHLYgyDMRLIHWCU5JMnmZ4rVTCG__jEWSZyUllTG8L6dJWOhCPFuR5V65afj4ShOMi3kaQYJSjyOsiOeDt63uetyBzNqcux5Bndt07B1zzU93fMd4tq-QA4WDoTREvRC8HAIgtdra-JeIpSK8TCt38ouvUmJN8fv2DbBAtu-jR3BINt0TcsgHvYQsxn2ie371CRmrhHvFDpYdpoFkapyAa_DsK31B_Fd17uVhLmM4zCdjshHIxF5k68Kx2UCSson7sGV9FEyIb9oQy4L0Q-5wQE0PzGiNusYwG_2Z-cnwRbCHQPmqb5ZxOrolBIEXyi1SYcYBsHwBWOkkw41kCG_ECQxQIgiu9kWQR1mUWrKJgNTwEdA6-GbxcwOgRRSAjHdwKwDtkC35DdLR6TDIOqUUMTUkdnRIeU3_mxrWZCFIk8vK8V1yXrS1qIYyCUjEFhBB8NCOpCGELuTT4Rg26IdYhrM6OSDwDqsfFW6mjTr6LqOUceARyb8JbBmCBwJ7lBsGQbAMli4gQizOyYx9I5uWwYGINuCQSyMrQ4BIFiLmjWFEeEjrA2gTZgAsuApg0WbHZsgA8YHUGgEOesomhqIUqAY04GizDZkF5ip1WGUMniCCOlgoCf8VUQ3QFhgRTB_A2aLbfhsQ0IPQMSEPha2AMjAGFqZTgELBlDdZIh2FP1hTKOjM8KAZ5IFVDfUrGW7ARPRgVu4k7NHzg9QGMA0oJ4kmE4Y7gApYGUFbyFJgOdAfSAtzB7WrSQKpszkExsBDYC0klaGbclJAcGUmAFrGNBaSg8zkI0kwU2jY8OMOwToBiSwbAN4hXUm-WnIeRE1exOoZGAL1gcjmJYkJ8yIWrZtwwIkWsO0JX0xcB9YhHHHRnrHsnU5A8vCcpkWUENKOJBJSgcjOgyuA12wDQoAwIYF3DYN4DnWFVuRhaUAAR-ALwiEDHrDmmA6llwlwOfEtICYAAGLgcTPAApiUHqQKik3MFVQHGLC4IBHB2KaIEgEpBVEiMCiQCqlbGBCYTCdIFtSikkoBMtgUkmkCZZlRp48dKsKAbYwA_0vbWURWuxSyWZDmYJ_vLu6-Kg-_GOQZaP07c7O3d3d9ohng9gX9xDjb7vxcGcEQGBEtlQEsCXR7nThf7s3Xw797te7D9_1XnTw-43T_fbt9seWNYh8s_flWNi2T673nIOYjpKHs-zb0cGWk325PBhff-Xjzz-S_aP0wxB9G19d-fzs4Zwkl6f_ubo4PDF3v335z9m3z-mdeZXtnolzcZwcnl99-Z0khEQnZ-np4Gg3S9HhF-fSH33odnHop8dD_dBzL_aMC3rs9k597-A20f9w2D2_tu9PQ_ebdWqd_r573_92fS7-6JkH46-33-KvQXg1uvx6vJ95KY_Zxef-2dcvIrrexRf9u0BP7oP0xOh_-nD9lV4bh_FWOuzRb13qBkZMrnt3f_yGev2Plz1yefTH2berq4f0wjrNgjtTbHW_33Q_b0WnR-RusP_tQ__mM947OkP3n45HXz9idPNpwK9_27_zj7BzeXxy_sP2PnTvDi67n7q7n3_74_ul9cG9t26_3UbZPv-RRddAl32fJodHZ31Y762eHdxaD3fXez5OPpxf2za-Gm3d3exnX76i3t3de8XcnQl33-X-PM1ZXXxTBlPJgtSftgZx21BJThFtPO6obGipo1HXUdnEUkda11Ep-qzjBOG7nfKE38nwOQnAuucxzY50YPWebOoolP_rw2h4h2g_wVr9ucRt5EqudL9eqaIg_EvUqif_XczCnc2QhGo_9YYcU3KoXXbtp_oHN3SZ0uWnlo44hJoQh_79qfJuR65bfjiFhErZHvlw8uWdSoNTSDGS7FAM090HSI0PZDFvbnu5CLllb6CholMFpshLILr3-TiUz38b8zCQ6QMqP_2Yn0uJ4mQ43XICVJAByEJPjlGmPsCA07OLk-7HvLEbZgU-OeIkI8i5XsypGkoHipcFoVSqL4-fdGer6PHQTVVIEURuOAbTHRWVgVLC48i5yoM3MrfzJmtMw1ieoRDhUGTtebyTjATGhakWoIdh7PCQzDDnVCBAmpvcS5yIjHuQz-4cg9tId-QqdxQ--PRpHEZy2yaQ65fZsMoXJdRci5Ib-XxPgIgHSVv7UWJC8TlvV_xoV6aHJ-ubW67KJxWlIj4ssq7JADubWuqU-arClNcZ08payw1F99KKiydV6NdavTw8IpIgutEu47E72BwVCoarUpQqgFQoUH48Xbj6rlpeb71dyMzTLOGhdp7AcG4WJ7OgRk7k76Fn-V7Gbsg9UaF75fmU8JWnU8qTDZM-H0XLh_nLaA7zvJX1vbVoniNY07RxHh5Jj101a7OnU3KXnr2WmMshtHyMpzVaTewkiMDpz9F8PQocgf0OvTO_exNHVeM-1zKlxKLnpYmoObwWifKxZSn88ehzM-uPIBpJzkOeCW_jFnIvSF152DGqCk758cwZlh6-mj-cjbHppfYzIUJYTFRZaenpdKG_jQP3--FYbVzPDJK-2ZXOBt6ENSpsxlJj9FnwURxp-K9wAWqnp0po-QTCz9_Lwcbs2ZTOqm41203CG6Z67Q7U5iOwnN77ocI_2YOu0GRhj9qYrAbfa6nidCAtH0m7g2hNm8SEmyeXOncN0VHwvUqk0vNa0lRgX4sg5UE2v_xPMVj4BZnJ9HHt4suQr-bSZ2NsfulXkQNDet0glAJXzVbm2mqJMAN-tRCymIo2HWrzlHh05GpB_FxqraXGAjyv5siKyDof5i-LrHfjOEv_Ck92ngg3mJyzmrKi9HRK-8mR1m7kyXskaRe-vZ5Gzmaw6UDpI_iZfsirOfPs4XS9s0evtUY5gqaG2LyqHaj6Zu7OKistN9QqWBX61UxOPsyc0_0ltAuvJGq7YRx7F_NhUenplO6lZ68lbWoITY3xiNDFx77IFtZKcy4U20k7utpQ6sg9FFk-nRzQqQea32GQUPRJqGr9PfAmB3uKuaqytJIEhec4Go0ztdr37SFkWdfO2PfllUF5OCtRtyH3Dw72e5fHn_aLocsgihHX0XjoyHtx-b-zk0R9oc5VaunYSfOPEDIE4k5NZA8EIAhTSfQw5KNUTE-iqEkXCwwBbgk21esomF40XIxr1qEe0_69SDJY72eeuEkgauc1bX9iUvmA8tCyPE9Vh03e4KtHlCt4T54dUuerayilrk3WY1Fepw5YNi6BPR6OeFg7ctH6BCUyud8AqhT4gSvPoS1nudydyHstocv0lkgtv4vDrPU41JXIOgR5Yz1wfrWxDrpoXUJVdamylqp5az34nnB57drzxnrg6ZH9OFKHMRdjmfZaguk0jpSQg9LMwuzF6CBprIbHCxGeybJ_sbNYh-kEbNSky1LFSQJnnNWrcanHElqpC0Y1FJJt9aD57ZmaNci2JZaocoGkhqDlPvWo8lsWtYZsGWh-CrmWfsWR5iUsKM7u15A_b11ChMkthpr1F81LlETZ3-5tHHj5sfYadZnrtsxgxDKoeykadVb_5WjmD--_HONByNPvtfwuWuvBr_I6wAIseWi1EhKpVC_DIHXrZRgu5gOJGezF8hBiem57IfCkdZniF8e5n40hP3T-bHB1Jv7Z0Mp8Q7gt5HbmMvs97bNEvLNxtAfEyJaI9oqo1LQW24HZ6tbClXuyhStdG2Oun8VF2WUqnHd5AhG44qMlwd5qmKYXSo4ED-VrHOLwZQgfXQh-0TohBeWRp274vXCh8lLheATIJjM7WxStz1g6j_XdziSPUufmZWZTXG_oZ4lMov-I4-EX-boHxOQ1cWphmxI7f16c1KHF6RyIePcCIHyiBGYyquz4-_v2lkGMbcuwkU0ZLm7WvlN5c3FwSH6enBsapyJ_j0C-MaAel87zyK5qNcXRqQtIsLOHt9rV6fFvV_utywHPtM-DwB1on3mqqcW2IEkYQkSr_Ufe82hdRcGPsdCO995q3MKYUcfykefqNrJ0eeLZojbXseAmsxzs-ZZp-zojhuchgRB2qOMh4rhY17nTUrNRpHirWUZLXUgRnpbFbzXcgvQiDFyQ57caar3R4alW-H1Ix2Wupo1EomGtkC-tKDnmUZEGvfZ_jIPRCPDlhxxaF2IoN941Fd5L-LuBiLSHeCyJpnFN-YbW_v0oTseJUA1B5MMcMnlXIoSsT-ORJu6zhGtbxj_lhMAgaFwpDAyTB0XSzLROQQ6cBzmloQIDdL0BpDfCUyWEE1m1iG6Elsi_79tom5Vz9WU99Cd7TGoEqrRSZTrWq1y_6F7st_YSfgPUOktkTh9lrU9BEngBX8Rv3_cs-MN8kGhuGL7lOoiaxPWRoBS4q9uOZTFGfM-hpsV02ySW47nMthzLNawqv-c4_NOXd1-A8N6fBP1T1qgSwUEFKxtNeW7Vwka5w_R2XtF1rnl3nJQbzXKbYrmWmzxt4nFeyiLyykykC5jY4xl3w4d0qO2BFCaiXmmFK3TfMS3KEWO-Ybk2RR7nVFjAP04NG3OKseEjwpjpC1BhA_nccnVhWi6xaZWJepmJuFXw6lxEArQE0isN_7PEn5KGPMFCq9yaF0I0VenI5aB_Fyde2nqDmVLDhSM8KUW9OEnGo-zlOrlBhi_pwZaIhLFAJHYTWbnsy3MTmUhaJ2DzwEr2wrGqHj-SC9NmXEcgFNyzkM10LmzP4MJwEdFtn-hAe8NzMcLCQzozXEMH_XdM07N10ybYnJcL9elC_HirMVoWEr310024D2T_s-t5qUa18_xgrpYXKGfNWIMOwLpKuxaAOc1nr-VndCv4oHt_yKFbFal2A9TMQEJJWSbUecaJZMhanSbdf2sJGnCk8QRX7pK60GlWTVgZNpdbeXm5ohR1K94HlkUcVjyWQQIIen1XiJ09MOPqOIMsBv1Swr2kh_FkD3OJgizQjzMnDG5lNPE54aNWn3vABe0Q_HarCDNAXExkTr7tgsgULx8JQgEea9tGlLUqUQdIt2m1ytWZBVBU18t6x7hLHdPAjOieMCwfc4_ZvscEtk3GBLGJrfvM1l1hu9R3qGt7Fny3fV_HjsUeBVHFSe63GkGt4hD7W223pI7SD8_UkbQg3Atl-FPMScYsD1KQAhUbJdq5fPUKSGdHm17iTrUBB5nD9J_ytYTQJoX5IE6ywH8Ao6xkm6sdqDw0m4k42GwwHo9juZallzVz0gjReTWua72xkIQeLgr6Wm8ILTfKegJMx673EVPTMfEk8nUFatTcNpQn_vfSkiU9rHo9Iov8jBhAYA1udTfhgdfqyk0grTsch2CaS3JumdiH9MAESbaITx0HOwaitnQkrq-bgnDOLdu3dIJcyCU4Mm3P0SGHQB4hTPcrcm4aZYHGVYGe-YMwjF0VhVzG94ELplEe05EiqGQIvoroJhuUQ883FFea3lBWL264EqUs7qNX4k1Zn9LUm1xEHmtK8QcZJSXpWySepKodSw5hTbzof5vVf548z-VDJ93D417rXEpDILkjkv-BJDr931RTpz_TIJRBQp40gH-V2d8exA1VMdexqfuUCZcIsM9g15GNiGHKmMo1IIaikA9Rxn0T0mQLjLvrYk8Qg3LM3PmcmMzMOSpLPKuG3VM5uZLiodLaomCngchAMu-Pw7BFwDAn8m03eeijmmUKDdlvC5u1Abc3Vpcx8iTpFfMauogXu3GWxUP5PgqIfcLRYDxH_g_8O3CozABkGJzLa9-MQ-6CPJcyAwFhHWZShi1u2zalHkauSRCmxIKUx2cUO9zFkAPZbtXO6BV_WoppSY3NWcqBHnyW5qgXR6lwZYLkqW3-SNUrALSlW3MBKqDQJq-_ajF9xsEiaQU69ORbMVKJQZYsXoNdz8tK2CKG7t_zYVBSrP4CneqDAxaJfLNrJXFFgmHThFyEMcMgns88TC2fY8R8GTxxhyLhO7bAzPLBg1DsYt1gjjAcagoDzTuQGsai56iWXs01J4UEZiziV38AYdlrsmsRM8xFlZ8x0BzSxLvWSSBDHpl4z8WzhDwVz1qWQctsgmRRUJvJ-JRQCyHXsWyOMLJ1x6A-MU3u6TbybMuxbVMX1LdtZDpUmkkGuadXZRNZHM8ebu1uXVSjWlZVSBmbPuaIultRbEWrEmFeoGvJGHHCKOWj5RtTVef8kFU1RKD1AeUb3VgWntqPw1OqL0wF35C8wjgfh84E0qzkstWRGu-_ive3FtbA-5DbJHEUiYmJupBCliv0SQyWO4PAQLuYs04mcqllU-GZvsEgwOUUPDzkdZwYBvKZa8t3gYKEO9gXHnUopxAHgKGyhG0wZDj14a0-Z5HeEH2JiBmmbDyOMhGGwY3M21oXUlA1ZWbkQb632hewOy6PojibxaJCyf3jvqUOIJCgEdUyu4wf8uvbj0Ehqq3kamqAj8Kvn4ub-7s4KQ9bXeA8fKkjRbnuyjnleyCPes9PqdGa9bXGXlZLOed3rW4CDFXFFHXRcd6l2OZTHoWAMS97FGyA0ui6sDExdZ1jiOFcKkzDp5bBdeG5romowxhkiciwfcP1DNeH8JvbHuKuWxtSVz2K8illj4KeWyeR2dysLpJXm6W3OQqy1mkcbamrkHkyOikO5j8joWENPKBKPaBHNNtEeqg4IGuRG6GP9P4NNR-p3xvdXphpgt9a6IZm3ga_mrf5W2gFRlW1yN-v0CrfMNYu72JNvuZaeGXx9gl1bS487GAL67rhgxexbSxM0yA2cS1d9wzbhK--SSzT8piNTeIS27a4hy1HVMQbhLaUMU6l-_PW9L_qrlvNYhbVS-XxF-1SyDMQrV6oCj4irJR4mOdixwPtNT0fe9h0bAaaSA1HR0zmvxDncSS4pSOqI2JBTE4tj0D-BXkatNhVRWXlqNys-sDq7lGNK9StJX7SXlAppNZjfcGL9eINJatUFfXFdZ2mJC_fKb4oPewLEDC1D5hfxZSMmSWH3TAUt0EeipX3pZlAxOO2yVwLVMbDQoAvgPze9mwXUn4DZMuAVF83Xd23Pcp8YiFOLZO6GJliLuVANZlh1R-gllFK7rpDeXVgkqhLjJFkdTZL3Q_lxk-qHQ-H4whwS6mYnkPwIeDBqIijwKH4siQjk03pcCa9FmGY7mBq-TWFpYim5xTAp8wB_mUCuVAQ2AJj0_8OabW2_yBaYDS3-gtDbmpBBI186kHibzLfwty0GSVI2L6OkAwRIPk0kQ5iQU0uMKa-a-qmzxiVWyzm_HGEEqvpXMhdtgJyd2CR2qdgFIzHThhsttonLA6tT_Y73pjmgq0KNt06mdV4yyNI771eHXhmi4jZRL5LDZK5OB2E-O4yEd9b_UwdiNPUNajZHiGFiLJ2i5AS3ZyPfzF7IgCuuFQDHKbjOp7tMcNChHKD2ph6AmHHwaawXMtF1Ceea4Btw8w2HZ16vsMQ9j3H48tMWyn2vdg61A7LCkDmjl8RIoVnT9xnQlJn8qAi6paxaAtvIu9Yx_XyV62fTV5lnRegW5NTVL08TVSXEwrHkMRDGXz_ygc45KWt-kN7kwteio3qKpn2E08vdhW3_SAYUy9EPHeuLj7Kk4f57azZqx2mr1VEk0nWwciQTus6D2nKJ-crNDJD8CT8ZMx5HGx9HGQDOLDWv-OjeUTGBhZkbGBBC3EU79EGBPMvx8yR5UdfSu_LnAz2FGel2Zqfgb7GKvJTlPMY8BoYFgnXs-i4iKlsA0zFG8BB1yRISTvxi6R6E2q6Go781PHklHKOMY_RS0zQnyWQ5IXipL9wASXu0afGz8tbL5GhxSplrI1hBmA-BZAvs-wTyDLTU7zpt2x6Si__XdH0LCbUOgqbHyeZAZDnCri-AbOtr2i2i5cfl2lXeh_yi2hHXyjmJTVjT81Apl1lAMzWstNrEHzyYrwJgP0sE8JeSJuX-0TyYgwvn8NabmgRGemmoi19rZl4D1p-O3TzdpVtKtTAm1kReSmLjHURlJTYfJ7hMTazdLa2VXiJVCkLVmKa9Sxq4w24jo0pFdsMH4wXhlv0xXx8acC3IV3UN2UbNoboOYnR82hRsgovkskNseLZOvJCSdqYeyArxofFT1mU48PSr1usGB_WWRqyAWuFN0WRjZm9x4jyatrkFUyrF9QWvMi-KaM1ZbSmjNaU0f6qMtovWDhrSmVNqexvWSprimN11GjKYU05rCmHNeWwphzWlMOaclhTDmvKYX_Hctj0fJl6UfnsNeV07eJYUxtramNNbez_aW0M_dqlMfTrlcbQr1gaQ2uXxtAvWBlDTWXs-ZUxtG5lDP1VhTHUFMaawlhTGNtMYQz9F9fFUFMXa-piTV3scV0MNXWxpi72X1YXW_RrfHpTHGuKY01xrCmONcWxpjjWFMea4lhTHGuKY01xrCmONcWxpjjWFMea4lhTHGuKY_8NxbGfWjrirrgUafZnu_zbD01prCmNNaWxpjTWlMaa0lhTGmtKY01prCmNNaWxpjTWlMaa0lhTGmtKY01prCmN_UKlscuEe6IveOIOPgv5EyBpMVjGM-1OPTkZh9n7Nt5GqK2F3BEhhH3yp0z2zvttLYV--Xf5dacGGG0bM-DK7yRMMJR_PmIJGlZCo378JbhVP9-uncdxOMF1GWc83D86n6x34RpzMqhPvTjygxvV9zzkrhjEoSeSgrpC_laZ-tUGoPdY_hL4-7aJJrSs6d9Lgqynfi5tBsSWg5wPHtLA5WH-uyczMExs-vRg81D6ExPM-1_EYah-2KG8tFpAyBmzYBQGIvkcyJ_MEl5f_ixLT_5OUF-EfmnOT6x1YkUn_SllNl4OUvz2xgzG0i2rFiQNboLwzD-P70QCk7yRSdF0cgrq3c6E6-92znk2OPN3x0Eof5XoX63_A4HNSfs=</pre>

### After screenshot:
loadouts

Updates:

Help Section
image

### CODE REVIEW

File: src/Modules/Build.lua [hunk 1]
```diff
@@ -226,12 +226,109 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
...
-		self.spec:SelectSecondaryAscendClass(value.ascendClassId)
-		self.spec:AddUndoState()
-		self.spec:SetWindowTitleWithBuildClass()
-		self.buildFlag = true
+	--self.controls.secondaryAscendDrop = new("DropDownControl", {"LEFT",self.controls.ascendDrop,"RIGHT"}, 8, 0, 120, 20, nil, function(index, value)
+	--	self.spec:SelectSecondaryAscendClass(value.ascendClassId)
+	--	self.spec:AddUndoState()
+	--	self.spec:SetWindowTitleWithBuildClass()
+	--	self.buildFlag = true
+	--end)
```
Comment (Paliak) [inline src/Modules/Build.lua:234]: I'd say add a comment here and in the other place related to `secondaryAscendancy` explaining why is this commented out. May not be obvious to someone trying to contribute a few leagues in the future.

Comment (Peechey) [reply src/Modules/Build.lua:235]: Added

**Review by LocalIdentity — APPROVED** [review]
Should be good

**Comment by Nuttar** [comment]
How is the list sorted? 
image
https://pobb.in/-eQbyFLKk95T
I added the {#} to them, hoping to just scroll through as I practice leveling but it's not particularly useful sorted like this.

**Comment by Wires77** [comment]
The list isn't sorted but probably should be.  I'll see if I can get that in before release

**Pull request merged**